### DISCUSSION TITLE
Connections in Radarr/Sonarr etc.

### DISCUSSION THREAD

**Discussion by saschabrockel** [RandomNinjaAtk/arr-scripts#108] [Q&A] [answered]
I'm confused. If I remember it correctly, the connections were added with your old repo where all things were included (`radarr-extended`).

Is it still the case with this repo and this manual? I guess not, right? But I found no manual anywhere that these need to be set. I just realized that I always get errors from PlexNotify.bash in Radarr in Sonarr and was wondering why. The path that was set was `/config/extended/scripts/PlexNotify.bash` but it has to be `/config/extended/PlexNotify.bash` now. That's no problem but that is why I was curious how it is handled now.

Or are these Connections not needed anymore? Because I didn't realize something wasn't working.

**Comment by RandomNinjaAtk** [comment, marked as answer]
AutoConfig automatically configures these for you. However, if you were migrating from the old container to using these scripts as a replacement, things were slightly tweaked and moved around, which means the existing connections won't work due to the changes... So to resolve the issue, if you remove your existing connections and let autoconfig run, it will configure them again correctly..
